client/relay-chain-minimal-node/src/blockchain_rpc_client.rs

```diff
@@ -0,0 +1,490 @@
+		Ok(result)
+	}
+
+	async fn persisted_validation_data(
+		&self,
+		at: Hash,
+		para_id: cumulus_primitives_core::ParaId,
+		assumption: polkadot_primitives::v2::OccupiedCoreAssumption,
+	) -> Result<
+		Option<
+			cumulus_primitives_core::PersistedValidationData<
+				Hash,
+				polkadot_core_primitives::BlockNumber,
+			>,
+		>,
+		sp_api::ApiError,
+	> {
+		let result = self
+			.rpc_client
+			.parachain_host_persisted_validation_data(at, para_id, assumption)
+			.await?;
+		Ok(result)
+	}
+
+	async fn assumed_validation_data(
+		&self,
+		at: Hash,
+		para_id: cumulus_primitives_core::ParaId,
+		expected_persisted_validation_data_hash: Hash,
+	) -> Result<
+		Option<(
+			cumulus_primitives_core::PersistedValidationData<
+				Hash,
+				polkadot_core_primitives::BlockNumber,
+			>,
+			polkadot_primitives::v2::ValidationCodeHash,
+		)>,
+		sp_api::ApiError,
+	> {
+		let result = self
+			.rpc_client
+			.parachain_host_assumed_validation_data(
+				at,
+				para_id,
+				expected_persisted_validation_data_hash,
+			)
+			.await?;
+		Ok(result)
+	}
+
+	async fn check_validation_outputs(
+		&self,
+		at: Hash,
+		para_id: cumulus_primitives_core::ParaId,
+		outputs: polkadot_primitives::v2::CandidateCommitments,
+	) -> Result<bool, sp_api::ApiError> {
+		let result = self
+			.rpc_client
+			.parachain_host_check_validation_outputs(at, para_id, outputs)
+			.await?;
+		Ok(result)
+	}
+
+	async fn session_index_for_child(
+		&self,
+		at: Hash,
+	) -> Result<polkadot_primitives::v2::SessionIndex, sp_api::ApiError> {
+		let result = self.rpc_client.parachain_host_session_index_for_child(at).await?;
+		Ok(result)
+	}
+
+	async fn validation_code(
+		&self,
+		at: Hash,
+		para_id: cumulus_primitives_core::ParaId,
+		assumption: polkadot_primitives::v2::OccupiedCoreAssumption,
+	) -> Result<Option<polkadot_primitives::v2::ValidationCode>, sp_api::ApiError> {
+		let result =
+			self.rpc_client.parachain_host_validation_code(at, para_id, assumption).await?;
+		Ok(result)
+	}
+
+	async fn candidate_pending_availability(
+		&self,
+		at: Hash,
+		para_id: cumulus_primitives_core::ParaId,
+	) -> Result<Option<polkadot_primitives::v2::CommittedCandidateReceipt<Hash>>, sp_api::ApiError>
+	{
+		let result = self
+			.rpc_client
+			.parachain_host_candidate_pending_availability(at, para_id)
+			.await?;
+		Ok(result)
+	}
+
+	async fn candidate_events(
+		&self,
+		at: Hash,
+	) -> Result<Vec<polkadot_primitives::v2::CandidateEvent<Hash>>, sp_api::ApiError> {
+		let result = self.rpc_client.parachain_host_candidate_events(at).await?;
+		Ok(result)
+	}
+
+	async fn dmq_contents(
+		&self,
+		at: Hash,
+		recipient: cumulus_primitives_core::ParaId,
+	) -> Result<
+		Vec<cumulus_primitives_core::InboundDownwardMessage<polkadot_core_primitives::BlockNumber>>,
+		sp_api::ApiError,
+	> {
+		let result = self.rpc_client.parachain_host_dmq_contents(recipient, at).await?;
+		Ok(result)
```

Comment:
```suggestion
		Ok(self.rpc_client.parachain_host_dmq_contents(recipient, at).await?)
```

Stylistic nit :P Could be done here everywhere.